docs/integrations/destinations/kvdb.md

```diff
@@ -1,4 +1,4 @@
-# KVDB [ARCHIVED]
+# KVDB
```

Comment:
The `[ARCHIVED]` will now be added automatically via https://github.com/airbytehq/airbyte/pull/35374